lib/scanner/kv-scanner/kv-scanner.h

```diff
@@ -46,6 +46,7 @@ struct _KVScanner
   const gchar *pair_separator;
   gsize pair_separator_len;
   gchar stop_char;
+  GString *formatted_key;
 
   KVTransformValueFunc transform_value;
```

Comment:
I can't see why the kvscanner needs the formatted key value. The only reason of moving this here as I understand the patch is that it is allocated on every invocation, but I think that is not enough reason to do feature envy and put a value used by a higher layer to here.